app/api/waitlists/[id]/users/notify/route.ts

```diff
@@ -55,29 +55,42 @@ export const POST = async (
     );
   }
 
-  let fidsToNotify: number[];
+  let UsersToNotify: { fid: number; address: string }[];
```

Comment:
This variable should be lowercased: "usersToNotify"

Reply:
True, I missed it because of auto completion and chat gpt ahah